lib/web_ui/skwasm/surface.cpp

```diff
@@ -176,20 +203,53 @@ class Surface {
     _surface->flush();
   }
 
+  void _rasterizeImage(SkImage* image,
+                       ImageByteFormat format,
+                       uint32_t callbackId) {
+    sk_sp<SkData> data;
+    if (format == ImageByteFormat::png) {
+      data = SkPngEncoder::Encode(_grContext.get(), image, {});
```

Comment:
A-ha! I bet removing the dependency on `SkPngEncoder` will take us below 1MB :)

Reply:
I'm not really sure we can get rid of this though. It might be possible... maybe draw the image to a canvas and then do `toDataUrl()` on it? I don't know.

Reply:
Yep, exactly that. I'm not requesting this change in this PR. Just spotted an opportunity for later.